.gitlab/ci/on-push.yml

```diff
@@ -17,7 +17,9 @@ trigger-private-build:
     - if: '$NIGHTLY'
   script:
     - echo "====Trigger Private Build===="
-    - python3 Utils/trigger_private_build.py --github-token $GITHUB_TOKEN
+    - git ls-remote --exit-code --heads "https://uesr:$GITHUB_TOKEN@github.com/demisto/content-private.git" "refs/heads/$CI_COMMIT_BRANCH" >> /dev/null && PRIVATE_BRANCH_NAME=$CI_COMMIT_BRANCH
```

Comment:
It's not clear what you're trying to achieve here. Please add a note, or extract to a script with a meaningful name.